updating/index.adoc

```diff
@@ -0,0 +1,81 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, check the following information:
+
```

Comment:
As i addressed in the pr #42047, `For more information...`should not be only related with `unmanaged Operators`. Maybe this [part](https://docs.openshift.com/container-platform/4.7/updating/understanding-the-update-service.html#update-service-overview_understanding-the-update-service) miss from the index here. I suggest we add it before `unmanaged operator`, and also for pr >=4.8+ versions.

Reply:
Addressed for this PR

Reply:
ack